maxim/logger/groq/helpers.py

```
@@ -0,0 +1,87 @@
+from groq._streaming import Stream, AsyncStream
+from groq.types.chat import ChatCompletionChunk
+from typing import Optional, Generator, AsyncGenerator
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Generator`, `AsyncGenerator`

Import from `collections.abc`

(UP035)